freqtrade/optimize/analysis/lookahead_helpers.py

```diff
@@ -19,7 +19,9 @@
 class LookaheadAnalysisSubFunctions:
     @staticmethod
     def text_table_lookahead_analysis_instances(
-        config: Dict[str, Any], lookahead_instances: List[LookaheadAnalysis]
+        config: Dict[str, Any],
+        lookahead_instances: List[LookaheadAnalysis],
+        caption: str | None = None,
```

Comment:
We support python 3.9 still - you can't use Union types like this yet (that's what's causing the test failures - at least the most obvious one)

Reply:
Fixed. Thanks for pointing that out. I'll be finished with this PR later today after my e2e manual test, and can fully open the PR then.